src/resources/Dissolutions/index.ts

```diff
@@ -3,3 +3,5 @@ export { CommunityContributionCompanyDissolutionResource } from '@/resources/Dis
 export { CooperativeDissolutionResource } from '@/resources/Dissolutions/cooperative'
 export { LimitedCompanyDissolutionResource } from '@/resources/Dissolutions/limitedCompany'
 export { UnlimitedDissolutionResource } from '@/resources/Dissolutions/unlimitedCompany'
+
+export { SolePropCompanyDissolutionResource } from '@/resources/Dissolutions/solePropCompany'
```

Comment:
To be consistent with the other SP resource (in registrations), please name the file "soleProprietorship".